src/ErrorDialog/ErrorDialog.tsx

```diff
@@ -25,40 +24,42 @@ const ErrorDialog = () => {
     const isVisible = useSelector(isVisibleSelector);
     const messages = useSelector(messagesSelector);
 
-    const defaultErrorResolutions = { Close: () => dispatch(hideDialog()) };
+    const defaultErrorResolutions = {
+        Close: () => dispatch(hideDialog()),
```

Comment:
Do we not use *camelCase* for functions, or is this a special case where we don't? 
`Close --> close`

Reply:
The `Close` here is being used as a label for the button text.